src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -75,33 +74,41 @@ class EntityClient (requestContext: RequestContext) extends Actor {
     }
   }
 
-  def createEntities(wsNamespace: String, wsName: String, entities: Seq[Entity]) = {
-    val results = entities.map(entity => createEntityOrReportError(wsNamespace, wsName, entity))
+  def createEntities(wsNamespace: String, wsName: String, entities: JsValue) = {
+    val entityJson = entities.convertTo[Seq[JsObject]]
+
+    // NOTE: this is very hacky, but we cannot support this quick fix upload well because
+    // we are accepting json in the Rawls model case class format.  Since we don't have access
+    // to the Rawls case classes, we are dealing with the json directly.  We need the entityType
+    // and name in order to show errors.  This will all be replaced in our next sprint.
+    def createFromJson(entity: JsObject) = {
+      // NOTE: these expect that entityType and entityName exist
+      val entityType = entity.fields.get("entityType").get.convertTo[String]
+      val entityName = entity.fields.get("name").get.convertTo[String]
```

Comment:
This line is failing because it doesn't handle the None case which is what one of the negative tests send (see MockServers.mockNonEmptyEntityFormData). Try 
```
val entityName = entity.fields.getOrElse("name", new JsString("")).convertTo[String]
```